app/networkapi/people/models.py

```diff
@@ -1,3 +1,17 @@
 from django.db import models
 
-# Create your models here.
+class Person(models.Model):
+    name: models.CharField(max_length=120)
+    role: models.CharField(max_length=120)
+    location: models.CharField(max_length=120)
+    teams: models.ManyToManyField(Team)
+    links: models.ManyToManyField(Link)
+    image: models.CharField(max_length=4096)
+    featured: models.BooleanField(default=False)
```

Comment:
let's actually not do featured stuff like this. We'll create a separate model for it so that it's easier to control via the admin interface.